docs/decommission.rst

```diff
@@ -0,0 +1,80 @@
+   - Backup USBs/other storage media
+   - Servers
+   - Firewall
+
+   .. note:: The recommended SecureDrop setup includes only one *Secure Viewing
+      Station* USB. However, if you have been working remotely or
+      have a non-standard setup, you may have more than one *SVS USB*. It is
+      important that you locate all of these USBs, since they hold the most
+      sensitive data.
+#. **Decide whether to save a backup.**
+   If you want a backup, you can, follow our `backup guidelines`_. Once the
+   backup has been created, you can move it off of the *Admin Workstation* USB
+   and onto an encrypted device, such as a LUKS-encrypted drive. You will also
+   require the *SVS* USB and/or a backup of the SecureDrop Application Key.
+#. **Disconnect the firewall and the servers from the internet.**
+   Be sure to inform your network administrator of any changes to devices on
+   your network.
+#. **Wipe or destroy the USB drives.**
+   Because the USB drives used for SecureDrop are all LUKS-encrypted,
+   reformatting the USB drives (in particular, overwriting a portion of internal
+   storage called the **LUKS header**) should be sufficient to make any existing
+   data on those drives unrecoverable.
+
+   For example, you could use your *primary Tails USB* to launch Gnome Disks,
+   insert and identify the USB drive you are trying to erase, and reformat this
+   drive with a new, LUKS-encrypted partition, erasing the existing partition
+   data.
+
+   .. caution:: Be **very** sure you are reformatting the right drive.
+      You may want to use the Secure Viewing Station laptop for this procedure
+      to reduce the risk of accidentally erasing a drive on your regular-use
+      machine.
+
+   You may also choose to destroy the drives by physical means, such as using a
+   hammer or purpose-built shredder to pulverize or destroy the drive.
+#. **Wipe or destroy the storage drives on the servers.**
+   SecureDrop submissions are stored GPG-encrypted on the Application server.
+   Unless your SecureDrop Application Key is compromised (or a significant
+   vulnerability in GPG is discovered), access to the servers does not guarantee
+   access to the submissions and messages you have received.
+
+   That said, there is still some sensitive information on the servers,
+   including system logs and the SecureDrop database, which would yield
+   information on the number of submissions and replies stored on the server.
+
+   Physically destroying or encrypting the storage drives on the servers are the
+   best ways to ensure that data on the drives cannot be recovered.
+
+   Physically destroying SSD drives is not as straightforward as destroying
+   older hard drives, and requires some research. Depending on your threat
+   model, you may choose instead to write over the information on the existing
+   drives. One way to do this is to re-install a clean version of Ubuntu server
```

Comment:
There's an option to "Overwrite free space" in the install process, this may need to be selected here, and would take extra time (a lot of time on spinning platter). I would suggest instead either referencing or suggesting ATA Secure Erase in the nist document mentioned above

Reply:
This part is the main part that I wanted your review on so I'm glad you're here :) 

* Good point re overwriting space in install wizard, I'll put that in 
* I hesitated on mentioning the ATA Secure Erase because the implementation depends on the manufacturer and sometimes that might not cut it[1]:

> Of the 12 drives we tested[,...] [e]ight of the drives reported that they supported the ATA SECURITY feature set. One of these encrypts data, so we could not verify if the sanitization was successful. Of the remaining seven, only four executed the “ERASE UNIT” command reliably. Drive B’s behavior is the most disturbing: it reported that sanitization was successful, but all the data remained intact. In fact, the filesystem was still mountable. [...] The wide variance among the drives leads us to conclude that each implementation of the security commands must be individually tested before it can be trusted to properly sanitize the drive.

[1] https://www.usenix.org/legacy/events/fast11/tech/full_papers/Wei.pdf

Reply:
See d89827e: I have included cursory information on ATA Secure Erase as one of the options for erasing the drives.